### DISCUSSION TITLE
Problems with Yarn Workspaces

### DISCUSSION THREAD

**Discussion by felipsbreno** [yarnpkg/berry#2617] [General]
In the company where I work we use the yarn workspaces, recently I have been having trouble running the command to climb the frontend.

Below is a picture of the error that is returned on the console when running the command to raise the frontend

![image](https://user-images.githubusercontent.com/50807243/111791979-8b6a2180-88a2-11eb-90ee-507834ea03c7.png)

Comand: `yarn run vigia-elite`
In the package.json file I created a script like this: `vigia-elite": "yarn install && yarn workspace vigia-elite start`,

Version: 
`Node.js > v12.21.0`
`Yarn > v1.22.10`

Has anyone here had a similar problem?

**Comment by merceyz** [comment]
This repo is for Yarn v2+, I also see you opened an issue at https://github.com/yarnpkg/yarn/issues/8598, there is no need to double post

**Comment by felipsbreno** [comment]
Ok @merceyz
